scripts/migrator_cleanup.sh

```diff
@@ -8,7 +8,9 @@ do
         esac
 done
 
-kubectl config use-context $context
+if [[ -n "$context" ]]; then
+  kubectl config use-context $context
+fi
```

Comment:
pleaseswap that with:
```
test -n ${context} && kubectl config use-context $context
```
Also, on L1 you just swapped `bash` for a posix sh, and `[[/]]` is a bash-extension, not part of the POSIX-sh standard.